src/clj_ldap/client.clj

```diff
@@ -200,16 +200,16 @@
 
 (defn- connect-to-host
   "Connect to a single host"
-  [{:keys [num-connections startTLS?]
+  [{:keys [num-connections max-connections startTLS?]
     :as options
-    :or {num-connections 1 startTLS? false}}]
+    :or {num-connections 1 max-connections 1 startTLS? false}}]
   (let [connection (create-connection options)
         bind-result (.bind connection (bind-request options))
         pcp (if startTLS?
               (StartTLSPostConnectProcessor. (create-ssl-context options))
               nil)]
     (if (= ResultCode/SUCCESS (.getResultCode bind-result))
-      (LDAPConnectionPool. connection num-connections num-connections pcp)
+      (LDAPConnectionPool. connection num-connections max-connections pcp)
```

Comment:
(LDAPConnectionPool. server-set bind-request initial-connections max-connections pcp)))

Reply:
ok

Reply:
@bskinny Can you give me some ideas on the difference between the two constructors?